xcm/xcm-builder/src/tests/assets.rs

```diff
@@ -33,8 +33,13 @@ fn exchange_asset_should_work() {
 		},
 	]);
 	let hash = fake_message_hash(&message);
-	let r = XcmExecutor::<TestConfig>::execute_xcm(Parent, message, hash, 50);
-	assert_eq!(r, Outcome::Complete(40));
+	let r = XcmExecutor::<TestConfig>::execute_xcm(
+		Parent,
+		message,
+		hash,
+		Weight::from_ref_time(50).set_proof_size(DEFAULT_PROOF_SIZE),
```

Comment:
I don't think we should be using `DEFAULT_PROOF_SIZE` here in tests - it is strictly for conversion from XCMv2 weight to XCMv3 weight. If you want some vaguely sensible proof-size constant to use in tests, just make a local declaration.